MASFoundation/Classes/models/Network/MASRequest.h

```diff
@@ -0,0 +1,147 @@
+//
+//  MASRequest.h
+//  MASFoundation
+//
+//  Copyright (c) 2017 CA. All rights reserved.
+//
+//  This software may be modified and distributed under the terms
+//  of the MIT license. See the LICENSE file for details.
+//
+
+#import <Foundation/Foundation.h>
+
+#import "MASClaims.h"
+#import "MASRequestBuilder.h"
+
+/**
+ MASRequest class is an object created by MASRequestBuilder. It's contains all parameters necessary to invoke an API.
```

Comment:
MASRequest class is an object created by MASRequestBuilder.  It contains all necessary information to invoke an API.